std_msgs/String class_name
geometry_msgs/Point32[] bounding_box
geometry_msgs/Point32[] segment